File: ecos-records/src/main/java/ru/citeck/ecos/records3/record/op/delete/dao/RecordDeleteDao.java
package ru.citeck.ecos.records3.record.op.delete.dao;

import org.jetbrains.annotations.NotNull;
import ru.citeck.ecos.records3.record.dao.RecordsDao;
import ru.citeck.ecos.records3.record.op.delete.dto.DelStatus;

public interface RecordDeleteDao extends RecordsDao {

    @NotNull
    DelStatus delete(@NotNull String recordId);
}

File: ecos-records/src/main/java/ru/citeck/ecos/records3/record/op/delete/dao/RecordDeleteDao.kt
package ru.citeck.ecos.records3.record.op.delete.dao

import ru.citeck.ecos.records3.record.dao.RecordsDao
import ru.citeck.ecos.records3.record.op.delete.dto.DelStatus

interface RecordDeleteDao : RecordsDao {

    fun delete(recordId: String): DelStatus
}
